### ISSUE TITLE
Export Document as PDF not working

### ISSUE THREAD

**Issue by KLynn2019** [asciidoctor/asciidoctor-vscode#284]
I have the most recent version of Asciidoctor-PDF installed and it runs successfully on the command line. Calling it from within VSC produces nothing...no save window, no PDF, not even an error message. I've tried this on several different Asciidoc files without success. VSC version is 1.43.2.

**Comment by danyill** [comment]
@KLynn2019 sorry to hear you're having troubles.

I'm using Code 1.43.2 and 2.7.15 of this extension and I'm able to render to pdfs using asciidoctor-pdf on an Ubuntu platform. I get the following with `asciidoctor-pdf -v`  within a console session:

```
Asciidoctor PDF 1.5.0.beta.7 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.6.3p62 (2019-04-16 revision 67580) [x86_64-linux]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
```
Can you look at the extension host logs and see if there are any errors using:

Cmd-Shift-P -> Developer: Show Logs -> Extension Host 

and paste in here any likely errors. Are you on Win/Mac/Linux?

**Comment by KLynn2019** [comment]
@danyill Thanks. I'm on Windows. I haven't been able to get this to work for a while now. Prior to my updating the gem, I too was using 1.5.0.beta.7. Now I'm on 1.5.3:

`Asciidoctor PDF 1.5.3 using Asciidoctor 2.0.10 [https://asciidoctor.org]`
`Runtime Environment (ruby 2.6.3p62 (2019-04-16 revision 67580) [x64-mingw32]) (lc:IBM437 fs:Windows-1252 in:IBM437 ex:IBM437)`

However, it wasn't working with that version either.

This morning, I opened VSC and opened the extension host logs in a terminal. I then tried exporting a document as a PDF...and it worked. I opened another document, in a different folder, without the logs displayed. Ran it again...and it worked. I have NO IDEA why...but thanks again.
